staging/src/k8s.io/legacy-cloud-providers/gce/gce_disks.go

```diff
@@ -515,7 +515,10 @@ func (g *Cloud) GetLabelsForVolume(ctx context.Context, pv *v1.PersistentVolume)
 
 	// If the zone is already labeled, honor the hint
 	name := pv.Spec.GCEPersistentDisk.PDName
-	zone := pv.Labels[v1.LabelFailureDomainBetaZone]
+	zone := pv.Labels[v1.LabelTopologyZone]
```

Comment:
FYI, We are no longer accepting enhancements on legacy-cloud-providers. Any such enhancements must be done out of tree. As this is clearly just a bug fix going to let it through.